projects/ng-keyboard-shortcuts/src/lib/keys.ts

```diff
@@ -1,6 +1,6 @@
 let isMac = false;
 if (typeof navigator !== 'undefined') {
```

Comment:
can be removed.

Reply:
extra space between, weird that prettier didn't fix it. maybe because we used the apply suggestion.